### PR TITLE
Creates guide for SNOW via #277

### LINKED ISSUE #277: Guide Wanted: How to use Sensu & Service Now
Expand on our existing documentation and provide a guide on using Sensu to create configuration items and remove them.

### PR DESCRIPTION
**Pull request by asachs01**
## Description
Fixes #277 and adds a guide on using SNOW & Sensu

## Motivation and Context
Guides clients through configuring the SNOW integration.

## How Has This Been Tested?
`yarn server`

Configurations have been tested against dev SNOW server

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] New doc/guide
- [ ] Fixing errata
- [ ] Update (Add missing or refresh existing content)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] All tests have passed.

### CODE REVIEW

File: content/sensu-enterprise/2.6/guides/using-snow.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    "cmdb_ci_table": "cmdb_ci_server",
+    "incident_management": true,
+    "incident_table": "incident",
+    "timeout": 10
+  }
+}{{< /highlight >}}
+
+Let's take a moment to walk through what we're specifying in that config file:
+
+* We three required attributes: `"host"`, `"user"`, and `"password"`
```
Comment (rgeniesse) [inline content/sensu-enterprise/2.6/guides/using-snow.md]: Not sure what this should be, We _provide_ three required attributes?

File: content/sensu-enterprise/2.6/guides/using-snow.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+* We then provide five additional attributes:
+  * `"create_cmdb_ci"` - this will create the configuration items automatically for Sensu clients. By default, this is `true`
```
Comment (rgeniesse) [inline content/sensu-enterprise/2.6/guides/using-snow.md]: Maybe change the language around describing the default so the can all be consistent. line below uses `here is` vs `this is` for all the other lines.

File: content/sensu-enterprise/2.6/guides/using-snow.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+     }
+   }
+ }
+}{{< /highlight >}}
+
+Let's walk through some of the key items in the configuration:
+
+* For registration, we provide `"servicenow"` inside of our `"handlers"` array
+* We set `"deregister"` to `true` to remove tell Sensu that we want clients deregistered
+* We the `"servicenow"` handler under `"deregistration"` so that our Sensu client will use the SNOW integration to deregister the client
```
Comment (rgeniesse) [inline content/sensu-enterprise/2.6/guides/using-snow.md]: We _set_ the `"servicenow"` maybe?

File: content/sensu-enterprise/2.6/guides/using-snow.md [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+- [SNOW Client Attributes][8]
+
+<!-- LINKS -->
+[1]: https://developer.servicenow.com
+[2]: https://slack.sensu.io
+[3]: /images/snow_inventory.png
+[4]: https://github.com/sensu-plugins/sensu-plugins-disk-checks
+[5]: /images/snow_incident.png
+[6]: /images/snow_incident_detail.png
+[7]: ../../integrations/servicenow/
```
Comment (rgeniesse) [inline content/sensu-enterprise/2.6/guides/using-snow.md:155]: This link goes to a relative path, but the last link in `[8]` goes to latest. Is there a preferred way to do this site wide?

File: content/sensu-enterprise/2.6/guides/using-snow.md (hunk 1 above)
Comment (asachs01) [reply content/sensu-enterprise/2.6/guides/using-snow.md]: Derp

File: content/sensu-enterprise/2.6/guides/using-snow.md (hunk 4 above)
Comment (asachs01) [reply content/sensu-enterprise/2.6/guides/using-snow.md:155]: Bumped this to 1.3

**Review by rgeniesse — APPROVED** [review]
🎉

**Pull request merged**